code/__HELPERS/game.dm

```diff
@@ -445,8 +445,9 @@
 		return candidates
 
 	for(var/mob/dead/observer/G in GLOB.player_list)
-		candidates += G
-
+		var/cooldown_time = CONFIG_GET(number/cooldown_antag_time) MINUTES
+		if(G.creation_time + cooldown_time < world.time) // Prevents people who have just died from becoming Antagonists
```

Comment:
Use the cooldown macro instead here

Reply:
it does the same stuff but is way better here for this usecase you can find it in the cooldown.dm file under `code\__DEFINES\cooldowns.dm` from line 63 to end use those instead.